alert-policies/nvidia-triton/RequestFailures.yml

```diff
@@ -0,0 +1,27 @@
+name: HTTP Request Failures
+
+description: |+
+  This alert is triggered when the HTTP Request Failures exceeds 1 for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered when HTTP Request Failures exceed 1 for 5 minutes.
```

Reply:
Hi @seemantk 
As per your suggestion, I have made change. Kindly review it